### ISSUE TITLE
music generation example

### ISSUE THREAD

**Issue by Jonathhhan** [zkmkarlsruhe/ofxTensorFlow2#28]
i tried to use this model for music generation: https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/audio/music_generation.ipynb#scrollTo=1mil8ZyJNe1w

its very rough yet, and not sure if everything works as expected.
maybe you can find something wrong...
the model is very small, so it is included in the example...
https://github.com/Jonathhhan/ofxTensorFlow2/tree/music_generation_example/example_music_generation

**Comment by Jonathhhan** [comment]
I guess it works now...

**Comment by danomatika** [comment]
Without trying it, I would say we could connect it to a synth with ofxMidi or generate audio directly with ofxPd.

**Comment by Jonathhhan** [comment]
@danomatika yeah, I already connected it with midi out from ofxMidi. Often it sounds quite random, but I can also hear some musical structure. Have to play a bit the the training settings...

**Comment by danomatika** [comment]
Looks like you need to track the duration for each pitch and send the noteoff individually. That might give you better rhythm and structure on the output.

**Comment by Jonathhhan** [comment]
It is interesting that the output seems to get more structure after running for some time.
But maybe that is because I choose a more or less random sequence for the beginning...

**Comment by Jonathhhan** [comment]
I added velocity to the model: https://github.com/Jonathhhan/ofxTensorFlow2/tree/music_generation_example/example_music_generation_2
And actually wonder, how to implement the music transformer: https://colab.research.google.com/notebooks/magenta/piano_transformer/piano_transformer.ipynb
